src/libs/auth.js

```diff
@@ -19,29 +20,110 @@ import {
 import * as Utils from 'src/libs/utils'
 
 
+export const getOidcConfig = () => {
+  const metadata = {
+    authorization_endpoint: `${getConfig().orchestrationUrlRoot}/oauth2/authorize`,
+    token_endpoint: `${getConfig().orchestrationUrlRoot}/oauth2/token`,
+    ...(isGoogleAuthority() && {
+      userinfo_endpoint: 'https://openidconnect.googleapis.com/v1/userinfo',
+      revocation_endpoint: 'https://oauth2.googleapis.com/revoke'
+    })
+  }
+  return {
+    authority: `${getConfig().orchestrationUrlRoot}/oauth2/authorize`,
+    client_id: authStore.get().oidcConfig.clientId,
+    popup_redirect_uri: `${window.origin}/redirect-from-oauth`,
+    silent_redirect_uri: `${window.origin}/redirect-from-oauth-silent`,
+    metadata,
+    prompt: 'consent login',
+    scope: 'openid email profile',
+    loadUserInfo: isGoogleAuthority(),
+    userStore: new WebStorageStateStore({ store: window.localStorage }),
+    automaticSilentRenew: true,
+    includeIdTokenInSilentRenew: true,
+    extraQueryParams: { access_type: 'offline' },
+    revokeTokenTypes: ['access_token', 'refresh_token']
```

Comment:
FYI: you don't need this line since those are the defaults

Reply:
Ah cool 👍

Reply:
Confirmed revoke works w/o this line, removed.